plugins/rocks.xml.webhelp/xsl/dita2xmlrocks_tables.xsl

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+                xmlns:xs="http://www.w3.org/2001/XMLSchema"
+                xmlns:dita-ot="http://dita-ot.sourceforge.net/ns/201007/dita-ot"
+                xmlns:related-links="http://dita-ot.sourceforge.net/ns/200709/related-links"
+                exclude-result-prefixes="#all"
+                version="2.0">
+
+    <xsl:param name="table-numbering"/>
+
+
+    <xsl:template match="*[contains(@class, ' topic/table ')]/*[contains(@class, ' topic/title ')]" mode="title-number">
+        <xsl:param name="number" as="xs:integer"/>
+        <xsl:choose>
```

Comment:
@Yuliia-Bondarenko-1806 please exclude all 'Table #.' string (or inline element with this string if exists)?